lib/cbus_elixir_web/controllers/speaker_controller.ex

```diff
@@ -57,4 +58,13 @@ defmodule CbusElixirWeb.SpeakerController do
     |> put_flash(:info, "Speaker deleted successfully.")
     |> redirect(to: speaker_path(conn, :index))
   end
+
+  # def get_meetings do
```

Comment:
any reason for the commented out code?

Reply:
That was the original place of the method you wanted me to move. I will remove it.